src/Core/src/RuntimeFeature.cs

```diff
@@ -80,10 +81,29 @@ internal static bool IsShellSearchResultsRendererDisplayMemberNameSupported
 		[FeatureGuard(typeof(RequiresUnreferencedCodeAttribute))]
 		[FeatureGuard(typeof(RequiresDynamicCodeAttribute))]
 #endif
-		internal static bool IsHybridWebViewSupported =>
+		public static bool IsHybridWebViewSupported =>
 			AppContext.TryGetSwitch("Microsoft.Maui.RuntimeFeature.IsHybridWebViewSupported", out bool isSupported)
 				? isSupported
 				: IsHybridWebViewSupportedByDefault;
+
+#if NET9_0_OR_GREATER
+		[FeatureSwitchDefinition("Microsoft.Maui.RuntimeFeature.EnableDiagnostics")]
+#endif
+		public static bool EnableDiagnostics
+		{
+			get
+			{
+				return AppContext.TryGetSwitch("Microsoft.Maui.RuntimeFeature.EnableDiagnostics", out bool isEnabled)
```

Comment:
Just a suggestion, I saw `"Microsoft.Maui.RuntimeFeature"` is repeated a lot, I used a constant and `nameof` in Android to make it more resistant to typos:

https://github.com/dotnet/android/blob/8ade56f7ee93813df58ccc0ec1acb765e72ffefa/src/Mono.Android/Microsoft.Android.Runtime/RuntimeFeature.cs#L8-L12